src/main/java/io/stargate/sgv2/jsonapi/api/model/command/validation/FindOptionsValidation.java

```diff
@@ -21,6 +32,28 @@ public boolean isValid(FindCommand value, ConstraintValidatorContext context) {
       return false;
     }
 
+    if (options.skip() != null && SortClauseUtil.resolveVsearch(value.sortClause()) != null) {
```

Comment:
Would it make sense to have `hasSortClause()` helper method in `FindCommand` to make calls bit more readable?